raaghu-layouts/src/rds-comp-app-shell/rds-comp-app-shell.stories.tsx

```
@@ -338,55 +916,563 @@ export const HeaderOnly: Story = {
-            label: "Users",
-            path: "/users",
-          },
+      <div
+        style={{
+          display: "flex",
+          flexDirection: "row",
+          alignItems: "flex-start",
+        }}
+      >
+        <div style={{ flex: "1" }}>
+          <RdsCompSideNavigation
```

ESLint

[error] Expected indentation of 20 spaces but found 10.

(indent)